### PR TITLE
Enforce value set

### PR DESCRIPTION
**Pull request by HugoIneat**
Enforce value set for now/later toggle button

### CODE REVIEW

File: src/pages/AdminDashboard/CreateRoomForm/CreateRoomForm.tsx [hunk 1]
```diff
@@ -77,7 +77,9 @@ const CreateRoomForm = ({ fields, onFormSubmit, onCreateFormReset }) => {
         event: React.MouseEvent<HTMLElement>,
         newValue: string | null
     ) => {
-        setToggleValue(newValue)
+        if(newValue !== null) {
+            setToggleValue(newValue)
```
Comment (MattiaPrimavera) [inline src/pages/AdminDashboard/CreateRoomForm/CreateRoomForm.tsx:81]: why is this needed, considering the state has already an initial value ?

**Review by MattiaPrimavera — CHANGES_REQUESTED** [review]

File: src/pages/AdminDashboard/CreateRoomForm/CreateRoomForm.tsx (hunk 1 above)
Comment (HugoIneat) [reply src/pages/AdminDashboard/CreateRoomForm/CreateRoomForm.tsx:81]: When you unselect a button, the value returned is null. In fact it disable the possibility to unselect a button.

ref : https://material-ui.com/components/toggle-button/#enforce-value-set

**Review by MattiaPrimavera — APPROVED** [review]
